unthink-stack/src/content/templates/version.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <style>
+    body {
+      margin: 2rem;
+      font-family: SFMono-Regular, Menlo, monospace;
+      text-rendering: optimizeLegibility;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+    }
+    .version {
+      display: inline-block;
+      padding: 0.75rem 1.5rem;
+      font-size: 0.875rem;
+      color: #305580;
+      background: #EFF7FF;
+      border-radius: 0.25rem;
+    }
+  </style>
+</head>
+<body>
+  <div class="version">{{ content.APP_NAME }} {{ APP_VERSION }}</div>
```

Comment:
You mentioned that `content.APP_NAME` vs `APP_VERSION` is weird. I would not have "APP_NAME" in all caps. It should follow normal language rules, which in this case is JavaScript (so camelCase). `APP_VERSION` is very specifically a global-level context variable, whereas `content.APP_NAME` is template content from your view. The all caps vs the camelCase helps with the distinction between the two.